Package.swift

```diff
@@ -0,0 +1,11 @@
+// swift-tools-version:4.0
+
+import PackageDescription
+
+let package = Package(name: "LaunchAtLogin",
+                      products: [.library(name: "LaunchAtLogin",
+                                          targets: ["LaunchAtLogin", "LaunchAtLoginHelper"])],
+                      targets: [.target(name: "LaunchAtLogin",
+                                        path: "LaunchAtLogin"),
+                                        .target(name: "LaunchAtLoginHelper",
+                                        path: "LaunchAtLoginHelper")])
```

Comment:
This is messy. Try to format it better and use tab-indentation.

Reply:
Ok